src/librustc_error_codes/error_codes/E0696.md

```diff
@@ -0,0 +1,52 @@
+A function is using `continue` keyword incorrectly.
+
+Erroneous code example:
+
+```compile_fail,E0696
+// Simple continue pointing to an unlabeled break should yield in an error
+fn continue_simple() {
+    'b: {
+        continue; // error!
+    }
+}
+// Labeled continue pointing to an unlabeled break should yield in an error
+fn continue_labeled() {
+    'b: {
+        continue 'b; // error!
+    }
+}
+// Simple continue that would cross a labeled block should yield in an error
+fn continue_crossing() {
+    loop {
+        'b: {
+            continue; // error!
+        }
+    }
+}
+```
+
+Here you have used the `continue` keyword incorrectly.
```

Comment:
Please replace "you" with "we". Also, this sentence is kinda a repeat from the first one at the top.